deploy/examples/monitoring/exporter-service-monitor.yaml

```diff
@@ -13,7 +13,6 @@ spec:
     matchLabels:
       app: rook-ceph-exporter
       rook_cluster: rook-ceph # namespace:cluster
-      ceph_daemon_id: exporter
```

Comment:
does service need to match all the labels?

Reply:
yes, the requirements (key, value pairs) are ANDed, https://github.com/prometheus-operator/prometheus-operator/blob/main/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml#L966

Reply:
example file is linked in the docs https://github.com/rook/rook/blob/master/Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md?plain=1#L59